File: output/qwen_2.5_72b_instruct/codenet/algo_based_translation/python/java/s993625131.java
import java.util.Scanner;

public class s993625131 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        List<String> S = Arrays.asList(input.split(""));
        int count = 0;
        int record = 0;

        for (int i = 0; i < S.size(); i++) {
            if (S.get(i).equals("A") || S.get(i).equals("C") || S.get(i).equals("G") || S.get(i).equals("T")) {
                count += 1;
            } else {
                if (count > record) {
                    record = count;
                }
                count = 0;
            }
        }
        if (count > record) {
            record = count;
        }
        System.out.println(record);
    }
}

File: dataset/codenet/Python/Code/s993625131.py
S = list(input())

count = 0
record = 0
for i in range(len(S)):
    if S[i] == "A" or S[i] == "C" or S[i] == "G" or S[i] == "T":
        count += 1
    else:
        if count > record:
            record = count
        count = 0
if count>record:
    record = count

print(record)